@xen-orchestra/proxy/USAGE.md

```diff
@@ -0,0 +1,22 @@
+## Reverse proxy configuration
+
+You can define multiples proxies in the configuration file
+
+```
+[reverseProxies.localhttps]
+path = '/https/'
+target = 'https://localhost:8080/'
+rejectUnauthorized = false
+
+[reverseProxies.localhttp]
+path = '/http/'
+target = 'http://localhost:8081/'
+```
+
+these two proxies will redirect all the queries to `<url of the proxy>/proxy/v1/https/` to `https://localhost:8080/` and `<url of the proxy>/proxy/v1/http/` to `https://localhost:8080/`.
```

Comment:
Introducing a prefix is a departure from what's being done in `xo-server`.

Although I can see some benefits (avoid clashes with internal URLs), I'm wondering if the current approach is a good one, these URLs are intended for external (ie users) usage, they probably should not change, maybe we should simply remove `v1`?

Reply:
I think versionning the api (v1)  is a good practice. Future us will think it's a good idea . 

for example when we will implement a web3 ( blockchain powered ) proxy :D

Reply:
Let's keep it like this and the future will tell us if that was the right choice.